test/test.js

```diff
@@ -65,6 +66,7 @@ describe('nemo-screenshot', function () {
         done();
     });
     it('will use @snap@ to take a screenshot', function (done) {
+        this.timeout(50000);
```

Comment:
My assumption is that this applies only to _this_ test? If so, then I guess the timeout is ok here, but should this ever take 50 seconds?